src/test/java/org/takes/facets/fallback/FbStatusTest.java

```diff
@@ -102,4 +104,31 @@ public void ignoresDifferentStatus() throws Exception {
             Matchers.equalTo(false)
         );
     }
+
+    /**
+     * Default response body should contain status code,
```

Comment:
@komaz wrong comment, http://www.yegor256.com/2014/04/27/typical-mistakes-in-java-code.html#test-method-names